_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -0,0 +1,235 @@
+---
+id: b8bb68
+name: Text spacing set in the `style` attribute is below permissible minimum value or not set to `!important`
+rule_type: atomic
+description: |
+  This rule checks that
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any HTML element that is [visible][] and has one of the following CSS properties set by the `style` attribute:
+
+- [word-spacing][]
+- [letter-spacing][]
+- [line-height][]
+
+## Expectation 1
+
+The test target does not have "important" as its [word-spacing][] [priority][], unless the [computed][] [word-spacing][] is greater than `0.16` times the computed [font-size][] of the test target.
+
+## Expectation 2
+
+The test target does not have "important" as its [letter-spacing][] [priority][], unless the [computed][] [letter-spacing][] is greater than `0.12` times the computed [font-size][] of the test target.
+
+## Expectation 3
+
+The test target does not have "important" as its [line-height][] [priority][], unless the [computed][] [line-height][] is greater than `1.5` times the computed [font-size][] of the test target.
+
+**Note:** Font size conversations rely on the default pixel size for body (usually 16 pixels), as defined by the [user agent origin][] stylesheet.
+
+## Assumptions
+
+If there is a mechanism available on the page by which text spacing can be adjusted, failing this rule might not mean [success criterion 1.4.12 Text spacing](https://www.w3.org/TR/WCAG21/#text-spacing) is not satisfied.
+
+## Accessibility Support
+
+While some assistive technologies are able to set [user origin][] styles, others such as browser extensions are only able to set style with the [author origin][]. Such assistive technologies can not create styles with a high enough priority to override a `style` attribute with `!important` . If accessibility support does not include assistive technologies that override text spacing through [author origin][], this rule should not be used.
+
+## Background
+
+- [Understanding Success Criterion 1.4.12: Text Spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)
+- [CSS Text Module Level 3 - Spacing](https://www.w3.org/TR/css-text-3/#spacing)
+- [CSS Visual formatting model details](https://drafts.csswg.org/css2/visudet.html)
+
+## Test Cases
```

Comment:
Unless you can show me that the default font size is 16px I recommend setting the font-size attribute to 16px in the style of the elements in the examples that do not use a relative size.